FiveCalls/FiveCalls/Issue.swift

```diff
@@ -42,6 +43,6 @@ extension Issue : JSONSerializable {
         
         let contacts = contactsJSON.flatMap({ Contact(dictionary: $0) })
         
-        self.init(id: id, name: name, reason: reason, script: script, contacts: contacts)
+        self.init(id: id, name: name, reason: reason, script: script, contacts: contacts, madeCalls: false)
```

Comment:
Here's another thing worth considering:

- JSON response -> structs -> immutable
- Local app data -> classes -> mutable

Then you just merge these in view models when dealing w/ the UI.  You have a lightweight set of mutable data that just refers to issue ids.